packages/snage/package.json

```diff
@@ -21,6 +21,7 @@
   "bin": "build/npm/snage.js",
   "dependencies": {
     "@js-joda/core": "^2.0.0",
+    "@types/inquirer": "^6.5.0",
```

Comment:
This should be a devDependency.

Reply:
I guess the same applies for `@types/yaml` in the next line?